assets/js/popup.js

```diff
@@ -0,0 +1,136 @@
+// Dynamic modal js functions
+
+// object element actribute generator
+const addAttributes = (element, attrObj) => {
+  Object.keys(attrObj).forEach((attr) => {
+    if (Object.prototype.hasOwnProperty.call(attrObj, attr)) {
+      element.setAttribute(attr, attrObj[attr]);
+    }
+  });
+};
+// HTML and chidl generator
+const createCustomElement = (element, attributes, children) => {
+  const customElement = document.createElement(element);
+  if (children !== undefined) {
+    children.forEach((el) => {
+      if (el.nodeType) {
+        if (el.nodeType === 1 || el.nodeType === 11) customElement.appendChild(el);
+      } else {
+        customElement.innerHTML += el;
+      }
+    });
+  }
+  addAttributes(customElement, attributes);
+  return customElement;
+};
+// Print modal-bg and modal content containers
+const printModal = (content) => {
+  const modalContent = createCustomElement('div', {
+    id: 'modal-content',
+    class: 'modal',
+  }, [content]);
+
+  const modalContainer = createCustomElement('div', {
+    id: 'modal-container',
+    class: 'modal-bg',
+  }, [modalContent]);
+
+  document.body.appendChild(modalContainer);
+};
+
+const modalProject = [
+  {
+    title: 'Tonic',
+    img: 'assets/images/desktop-images/desktop-work1.png',
+    description: 'A daily selection of privately perosonalized reads; no accounts or sign-ups required.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'assets/images/desktop-images/desktop-work2.png',
+    description: 'Experimental content creation feature that allows users toadd to an existing story over the course of a day without spamming their friends.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Facebook 360',
+    img: 'assets/images/desktop-images/desktop-portfolio-image1.png',
+    description: 'Exploring the future of media in Facebooks first VirtualReality app; a place to discover and enjoy 360 photos andvideos on Gear VR.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Uber Navigation',
+    img: 'assets/images/desktop-images/desktop-portfolio-image2.png',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  }];
+// Modal projects event
+// First Modal
+projectBtn.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    printModal(`<div class="modal-header-container">
+    <div class="modal-title-container">
+      <h2 class="modal-title">${modalProject[index].title}</h2></div>
```

Comment:
- [ ] Please add more padding between the modal description and modal image as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="1216" alt="Screenshot 2022-04-07 at 13 03 24" src="https://user-images.githubusercontent.com/77038610/162162871-ca257aa0-e2c2-43ca-91ca-53e135007217.png">  | <img width="824" alt="Screenshot 2022-04-07 at 13 03 07" src="https://user-images.githubusercontent.com/77038610/162162825-9226c2dd-93af-41ee-827b-b6aa4d8cd727.png">

- [ ] Please add padding around the modal in mobile view and make sure to center the buttons, currently the content sticks to the edges:

Your design  | Figma design
------------- | -------------
<img width="262" alt="Screenshot 2022-04-07 at 13 06 22" src="https://user-images.githubusercontent.com/77038610/162163457-8e653b54-29d2-499f-a370-8e1f3141fcc0.png"> | <img width="269" alt="Screenshot 2022-04-07 at 13 06 41" src="https://user-images.githubusercontent.com/77038610/162163523-8207f716-b46a-4be0-8751-41ff29613ae1.png">